system/help/help.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Toggles the visibility of the information icons.
+ *
+ * @returns void
+ */
+function toggleInfoButtons(e) {
+	$(e).toggleClass('clicked');
+	$('.hyphaInfoButton').toggle();
+}
+
+/**
+ * Creates an info popup.
+ *
+ * @param anchor Element, mouse click event, used to determine position of popup
+ * @param key string, the key for with to get the info
+ * @param language string
+ * @param info Element, the info button for which to make an info popup
+ */
+function toggleInfoPopup(anchor, key, language, info) {
```

Comment:
The anchor is the click event. Is `anchor` a good / common name for this? Also, `info` is the clicked element, which isn't directly clear to me. Perhaps `info_icon` is more appropriate?